tools/cli/README.md

```diff
@@ -0,0 +1,71 @@
+## What
+Cadence CLI provide an command-line tool for users to perform various tasks on Cadence.  
+Users can perform operations like register, update and describe on domain;  
+also start workflow, show workflow history, signal workflow ... and many other tasks.    
+
+## How
+- Run `make bins`
+- You should see an executable `cadence-cli`
+
+## Quick Start
+Run `./cadence-cli` to view help message. There are some top level commands and global options.   
```

Comment:
Maybe I can combine currently `cadence` and `cadence-cli` to one, and create commands like `cadence service start`? What do you think?
Otherwise we have to use `cadence-cli ...`

Reply:
Let's rename cadence to cadence-server and cadence-cli to cadence then

Reply:
Yes, added

Reply:
Done